src/App.tsx

```diff
@@ -145,11 +74,16 @@ export const App: React.FC = () => {
```

Comment:
The error messages 'Unable to add a todo', 'Unable to delete a todo', 'Unable to update a todo' are always shown together when 'isError' is true. Consider displaying more specific error messages based on what action failed.